scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -470,58 +546,68 @@ We can correct this by applying a "shrinkage" procedure, which will adjust large
 
 To do this, we will use the `lfcShrink()` function, but first we need to know the name and/or position of the "coefficient" that was calculated by `DESeq()`, which we can do with the `resultsNames()` function.
 
-```{r}
+```{r coefficient}
 # identify position of coefficient
 DESeq2::resultsNames(deseq_object)
 ```
 
 
-```{r}
-# appyly logFC shrinkage
+```{r shrinkage}
+# appyly logFC shrinkage using the default model
 shrink_results <- DESeq2::lfcShrink(deseq_object, res = deseq_results, coef = 2, type = "apeglm")
```

Comment:
This could go to multi-line as needed for spacing - 
 
```suggestion
shrink_results <- DESeq2::lfcShrink(deseq_object, 
                                                             res = deseq_results, 
                                                             coef = 2, 
                                                             type = "apeglm")
```

Reply:
Hm spacing isn't great over here in github, so you might want to let RStudio help!